stacks-v3/src/App.js

```diff
@@ -506,7 +506,24 @@ class App extends Component {
     }
     return today;
   };
-  dayController = () => {};
+  dayController = () => {
+    //get the day
+    let fullDate = new Date();
+    let thisDay = fullDate.getDate().toString();
+    let thisMonth = fullDate.getMonth().toString();
+    let thisYear = fullDate.getFullYear().toString();
+
+    //build dates
+    let currentDate = thisDay + "/" + thisMonth + "/" + thisYear;
```

Comment:
Perhaps this could be a template string?

```suggestion
    let currentDate = `${thisDay}/${thisMonth}/${thisYear}`;
```